### ISSUE TITLE
Consider write the documentation for instance.refresh, instances.forceUpdate& context.notify

### ISSUE THREAD

**Issue by p4bl1t0** [pocesar/react-native-stager#12]
You should consider write the docs for instance.refresh, instances.forceUpdate& context.notify

It was no clear for me in how to make an stage to react to a state change

**Comment by pocesar** [comment]
yeah, there's a shy mention of "refresh" at the bottom inside caveats, will edit it to make clearer
